kotsadm/pkg/handlers/license.go

```diff
@@ -465,6 +427,7 @@ func GetOnlineInstallStatus(w http.ResponseWriter, r *http.Request) {
 	JSON(w, 200, status)
 }
 
+// NOTE: there is no auth on this route
```

Comment:
here

Reply:
this needs to be addressed (not in this PR).  this function exists for backwards compatibility with platform only.

Reply:
@divolgin why can't we add auth here?

Reply:
@marccampbell wouldn't that break backwards compatibility?